.github/ISSUE_TEMPLATE/bug-report-advanced

```diff
@@ -0,0 +1,74 @@
+name: "\U0001F41E Bug report"
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before submitting a bug report**
+
+        The issue list is reserved for bug reports and feature requests. If you have a usage question, you can:
+
+        - Read the [documentation](https://blazorise.com/docs)
+        - Ask questions on [Discord](https://discord.io/blazorise)
+
+        Also, try to search for your issue/feature - another user may have already requested something similar; Thanks!
+
+  - type: input
+    id: version
+    attributes:
+      label: Blazorise Version
+    validations:
+      required: true
+  - type: input
+    id: reproduction-link
+    attributes:
+      label: Link to minimal reproduction
+      description: |
+        The easiest way to provide a reproduction is to provide a link to a public GitHub repository or a tool like [Telerik Repl](https://blazorrepl.telerik.com/).
+
+        The reproduction should be **minimal**. This means it should contain only the bare minimum amount of code needed
+        to show the bug.
```

Comment:
Maybe, Add some text about making it so the code provided is as complete as possible to get it running, i.e contain class definitions, etc...